README.md

```diff
@@ -10,6 +10,11 @@
 
 <image src="https://github.com/GoogleContainerTools/jib/raw/master/logo/jib-build-docker-java-container-image.png" alt="Jib - Containerize your Java applications." width="650px" />
 
+
+| ☑️  Jib User Survey |
+| :----- |
+| What do you like best about Jib? What needs to be improved? Please tell us by taking our [simple survey](https://forms.gle/YRFeamGj51xmgnx28). Your responses will help us understand Jib usage and allow us to serve our customers (you!) better. |
```

Comment:
If you're up for one more tweak, providing a time estimate can help lure people in

```suggestion
| What do you like best about Jib? What needs to be improved? Please tell us by taking our [one-minute survey](https://forms.gle/YRFeamGj51xmgnx28). Your responses will help us understand Jib usage and allow us to serve our customers (you!) better. |
```